test/functional/mempool_package_rbf.py

```diff
@@ -162,13 +162,13 @@ def test_package_rbf_additional_fees(self):
         self.log.info("Check replacement pays for incremental bandwidth")
         _, placeholder_txns3 = self.create_simple_package(coin)
         package_3_size = sum([tx.get_vsize() for tx in placeholder_txns3])
-        incremental_sats_required = Decimal(package_3_size) / COIN
-        incremental_sats_short = incremental_sats_required - Decimal("0.00000001")
+        incremental_sats_required = (Decimal(package_3_size * 0.1) / COIN).quantize(Decimal("0.00000001"))
+        incremental_sats_short = incremental_sats_required - Decimal("0.00000005")
```

Comment:
Why do you change this value from being 1 sat smaller to being 5 sats smaller?

Reply:
The string representation of the amount depends on its value. Was having `"blahblah 0.0002" is not the same as "blahblah 0.00020"` issues, so I just changed the amount. Yes it's brittle, but wanted to pick my battles for this PR.